app/lnd/config/index.js

```diff
@@ -2,10 +2,17 @@
 // Mac OS X: /Users/user/Library/Application Support/Lnd/tls.cert
 // Linux: ~/.lnd/tls.cert
 // Windows: TODO find out where cert is located for windows machine
-import { userInfo } from 'os'
+import { userInfo, platform } from 'os'
+
+var loc;
```

Comment:
let's use `let` instead of `var`

we also have decided to avoid semicolons in this codebase 😄